config/app_local.php.template

```diff
@@ -342,6 +342,10 @@ return [
         'shutdownWarning' => false,
     ],
 
+    'TermsOfUse' => [
+        'version' => '2',
```

Comment:
I'm wondering if it makes sense to put the version number in the config. It's not really a config option to me. For instance we don't need to change the value based on the environment where it's deployed. And we shouldn't actually be able to change the value unless there has been actual changes in the ToS itself. So it would probably be better if the version number is hardcoded in a constant.

Reply:
Yeah, I'm not sure what I was thinking there. What would be the best place to stuff a global constant? I'm thinking about adding it to `src/Application.php`, but maybe there's a better place to put it.

Reply:
OK, done. I didn't update the date for the new terms to go into effect, so you'll have to do that once you know when you'll deploy it.